### PR TITLE
Kinit

### PR DESCRIPTION
**Pull request by wouterhummelink**
Fixes issue #51

### CODE REVIEW

File: lib/facter/ipa_replicascheme.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
       if host and domain
         fqdn = [host, domain].join(".")
       end    
-    servers = Facter::Util::Resolution.exec("/sbin/runuser -l admin -c '/usr/sbin/ipa-replica-manage list' 2>/dev/null | /bin/egrep -v '#{fqdn}|winsync' | /bin/cut -d: -f1")
+    servers = Facter::Util::Resolution.exec("/sbin/runuser -l admin -c '/kinit -kt ~admin/admin.keytab admin && klist && usr/sbin/ipa-replica-manage list' 2>/dev/null | /bin/egrep -v '#{fqdn}|winsync' | /bin/cut -d: -f1")
```
Comment (robruma) [inline lib/facter/ipa_replicascheme.rb]: Sorry, this line still has a typo

**Comment by kallies** [comment]
What's missing to get this PR merged?

**Comment by wouterhummelink** [comment]
Abandoning it since I rewrote the entire thing from scratch awhile ago.

**Comment by kallies** [comment]
@wouterhummelink can you post a link to the rewrite please?

**Comment by wouterhummelink** [comment]
I can’t since it isn’t open sourced. My company doesn’t generally allow sharing in house modules,

Wouter Hummelink
Technical Consultant Tooling & Automation
Integrated Design & Delivery
KPN Service & Delivery
T: +316-12882447

**Pull request closed without merging**